Doc/library/array.rst

```diff
@@ -188,11 +188,16 @@ The following data items and methods are also supported:
    array of some other type.
 
 
-.. method:: array.index(x)
+.. method:: array.index(x[, start[, stop]])
```

Comment:
You can document the default start value:

array.index(x, start=0[, stop])

Reply:
I would prefer to specify `start=0, stop=sys.maxsize`, but there were objections against this, because this looks like *start* and *stop* are keyword parameters. There were issues about changing the documentation back to `[, start[, stop]]`.